### DISCUSSION TITLE
Synchronize Settings cross-platform

### DISCUSSION THREAD

**Discussion by RoSky7791** [coder/code-server#6861] [General]
Hi,

I am using the docker image of code-server. I would to know how to sync all of my settings/addons/etc. between my Windows 11 VSCode, and code-server ? I wand to have the same interface of code-server with pluggins, settings across my computer, iPad, etc.

Thanks for your help

**Comment by code-asher** [comment]
I do not believe there is any built-in method for syncing between code-server and VS Code.  You could try Microsoft's web server instead of our code-server (it can be ran with `code serve-web`), then you should be able to use the built-in settings sync.

But if that is not an option, then I think the only way is to manually sync the user data directory, for example by using `rsync` over ssh.

**Comment by StrangeNoob** [comment]
I'm not entirely sure how this happened, but it worked for me.

![Screenshot 2024-09-22 at 10 08 30 PM](https://github.com/user-attachments/assets/2fed2812-2b2f-4dc4-a3e7-a36e99077c20)

I zipped all my local VSCode extensions (~/.vscode/extensions), SCPed them to the Docker-mounted config, and reloaded Code-Server. It resolved the issue.
